.github/workflows/build-ci.yml

```diff
@@ -34,20 +34,19 @@ jobs:
                     - "11.*"
                     - "12.*"
                 driver:
-                    - 1
+                    - 2
```

Comment:
I have to change the default driver version that is tested to match the ruleset from https://github.com/mongodb/laravel-mongodb/pull/3347